pkgs/servers/x11/xorg/xwayland.nix

```diff
@@ -1,4 +1,6 @@
-{ stdenv, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config, epoxy, libxslt, libunwind, makeWrapper, egl-wayland }:
+{ stdenv, wayland, wayland-protocols, xorgserver, xkbcomp, xkeyboard_config
+, epoxy, libxslt, libunwind, makeWrapper, egl-wayland
+, useGlobalFontPath ? false }:
```

Comment:
Why not make this a string directly? It would make changing the path for non-NixOS users easier.

Reply:
done, I renamed it to defaultFontPath